### ISSUE TITLE
Python regression test sometimes fails on test_model_watch_service_endpoints_state

### ISSUE THREAD

**Issue by kirillmakhonin** [legion-platform/legion#408]
======================================================================

ERROR: test_model_watch_service_endpoints_state (test_k8s_model_operations.TestK8SModelOperations)
----------------------------------------------------------------------
```
Traceback (most recent call last):
  File "/var/lib/jenkins/workspace/Deploy_Legion_Release@2/tests/python/test_k8s_model_operations.py", line 169, in test_model_watch_service_endpoints_state
    model_service = enclave.deploy_model(TEST_IMAGE)
  File "/var/lib/jenkins/workspace/Deploy_Legion_Release@2/legion/legion/k8s/enclave.py", line 404, in deploy_model
    .format(image_meta_information.model_id, image_meta_information.model_version))
Exception: Cannot find created model service for model (id=demo-abc-model, version=1.0)
```

**Comment by ablatov** [comment]
@kirillmakhonin I suppose that "enclave.deploy_model(TEST_IMAGE)" method does not use custom or long timeouts for the model deploy creation as legionctl command uses now. Increasing default timeouts may help.

**Comment by kirillmakhonin** [comment]
@ablatov  Yeah, but error is "cannot find created model service". This is very strange, because it means that legion can not retrieve K8S Service (in any state) at all

**Comment by kirillmakhonin** [comment]
It looks like one master did not have time to synchronize new created service after it has been created. Creation request had been send to master A and at once  service list request has been sent to master B.
Kubernetes does not provide any API to ensure that `create_namespaced_service` operation has been completed.

Possible solutions:

1. Add iterated check
2. Add sleep

**Comment by kirillmakhonin** [comment]
Closed by #418
